game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleState.java

```diff
@@ -107,4 +109,8 @@ public boolean isFirstRound() {
   Collection<Unit> getTransportDependents(Collection<Unit> units);
 
   Collection<IBattle> getDependentBattles();
+
+  @RemoveOnNextMajorRelease("Only used to handle step string changes")
```

Comment:
nit, I'm not sure the annotation comment string here makes a lot of sense. Would we be just as well off if it were removed? Otherwise, can it be made more direct, is it indicating that we have to do something else beyond upgrading to a non-compatible release to remove this method?

Reply:
I've removed the annotation comment string.